python-ectoken/requirements.txt

```diff
@@ -13,6 +13,6 @@
 # * limitations under the License.
 # */
 
-pyOpenSSL>=0.15.1
+pyOpenSSL>=19.0.0
```

Comment:
No longer necessary?

Reply:
removed pyOpenSSL in requirements and comments , ty